From: =?utf-8?q?David_Pr=C3=A9vot?= <david@tilapin.org>
Date: Fri, 8 May 2015 11:44:04 -0400
Subject: Revert "Hardwire PHPParser 1.2.2"

This reverts commit c264bd7070a204d215be6ae8b38d910276ebed96.

Forwared: not-needed
Origin: upstream, https://github.com/theseer/phpdox/commit/c264bd7070a204d215be6ae8b38d910276ebed96
---
 composer.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/composer.json b/composer.json
index 8959801..b4a7cae 100644
--- a/composer.json
+++ b/composer.json
@@ -24,7 +24,7 @@
     "theseer/directoryscanner" : ">=1.3.0",
     "theseer/fxsl" : ">=1.1",
     "phpunit/php-timer" : ">=1.0.4",
-    "nikic/php-parser" : "=1.2.2"
+    "nikic/php-parser" : ">=1.2.1"
   },
   "autoload": {
     "classmap": [
